zrml/swaps/rpc/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 //! RPC interface for the Swaps pallet.
 
-use core::{fmt, str::FromStr};
+use core::{fmt::Display, str::FromStr};
 use jsonrpc_core::{Error as RpcError, ErrorCode, Result};
 use jsonrpc_derive::rpc;
 use parity_scale_codec::Codec;
```

Comment:
[nitpick] `MaxEncodedLen` imports appear a bit inconsistent in this PR (sometimes imported directly, sometimes not). Is there some pattern that I don't see?